dev/com.ibm.ws.microprofile.faulttolerance.telemetry.metrics.2.0/bnd.bnd

```diff
@@ -0,0 +1,59 @@
+#*******************************************************************************
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+openTelemetryVersion=1.35.0
+
+Bundle-Name: com.ibm.ws.microprofile.faulttolerance.telemetry.metrics.2.0
```

Comment:
This bundle should probably be called: 

io.openliberty.microprofile.faulttolerance.4.0.telemetry.metrics.2.0